### PR TITLE
feat: Add quick start

### PR DESCRIPTION
**Pull request by gaocegege**
Signed-off-by: Ce Gao <gaoce@caicloud.io>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/katib/878)

### CODE REVIEW

File: examples/v1alpha3/quick-start-example.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+metadata:
+  namespace: kubeflow
+  name: quick-start-example
+spec:
+  parallelTrialCount: 3
+  maxTrialCount: 12
+  maxFailedTrialCount: 3
+  objective:
+    type: maximize
+    # It is a hack to show the functionality of katib, in production we can use 1.
```
Comment (johnugeorge) [inline examples/v1alpha3/quick-start-example.yaml]: I didn't get this logic. Can you explain more?

Comment (gaocegege) [reply examples/v1alpha3/quick-start-example.yaml]: If we set it to 1, the experiment will be finished soon (with 1-2 trials).

**Comment by gaocegege** [comment]
Rebased the upstream to see if we could pass the CI.

**Comment by johnugeorge** [comment]
Strange. It passed

**Pull request merged**